TerminalDocs/panes.md

```diff
@@ -224,6 +224,27 @@ You can toggle read-only mode on a pane with the `toggleReadOnlyMode` command.
 { "command": "toggleReadOnlyMode" }
 ```
 
+You can enable read-only mode on a pane. This works similarly to toggling, however, will not switch state if triggered again.
+
+**Command name:** `enableReadOnlyMode`
```

Comment:
Are these definitely in 1.16 (which is active right now in Stable)?

Reply:
I just checked and they are not! :)
Great catch, Dustin! It looks like I missed this while I was stashing `enabledReadOnlyMode` docs in the 1.18 release branch: https://github.com/MicrosoftDocs/terminal/pull/645#issuecomment-1502434075
Turns out I only stashed them for `actions.md` and `panes.md`

I'll stash this into the 1.18 branch and update this PR tomorrow :)

Reply:
PR made to remove this from main: https://github.com/MicrosoftDocs/terminal/pull/672